tox.ini

```diff
@@ -1,8 +1,17 @@
 [tox]
 envlist =
+    py27
+    py34
+    py35
+    py36
+    py37
     flake8
     isort
 
+[testenv]
+whitelist_externals = make
```

Comment:
To me, `make` seems like an unnecessary layer for such a simple package and doesn't offer anything over the `tox` CLI. WDYT about just putting the test command here?